app/javascript/elm/src/Main.elm

```diff
@@ -565,8 +565,8 @@ view model =
         , main_
             []
             [ div [ class "maps-page-container" ]
-                [ div [ class "map-filters" ]
-                    [ viewSessionTypes model
+                [ div [ class "filters" ]
+                    [ viewSessionTypeNav model
```

Comment:
The `view` function is huge and a bit messy. That's my bad when unifying the Elm apps into 1. Whenever you can please make it better like in this case. I guess a priority would be to extract a lot of functions out